sound/soc/sof/ipc3-topology.c

```diff
@@ -2112,6 +2112,15 @@ static int sof_ipc3_set_up_all_pipelines(struct snd_sof_dev *sdev, bool verify)
 				sroute->sink_widget->dynamic_pipeline_widget))
 			continue;
 
+		/*
+		 * For virtual routes, both sink and source are not
+		 * buffer. IPC3 only supports connections between buffers and components. Ignore
```

Comment:
both sink and source can not be buffer.

Reply:
we dont have this case in our topologies but we have connections between virtual widgets.